src/desktops/dskGameInterface.cpp

```diff
@@ -689,6 +689,9 @@ bool dskGameInterface::Msg_KeyDown(const KeyEvent& ke)
                 {
                     isCheatModeOn = !isCheatModeOn;
                     curCheatTxt.clear();
+                    char text[256];
+                    snprintf(text, sizeof(text), "Cheat mode is on");
```

Comment:
Shouldn't this check for ``isCheatModeOn``?

Reply:
You are right. Thanks